site/about.html

```diff
@@ -0,0 +1,32 @@
+<html>
+	<head>
+		<title>About Me</title>
+		<link rel="stylesheet" href="style.css"
+	</head>
+
+	<body>
+		<nav>
+			<a href="about.html">ABOUT ME</a> | <a href="portfolio.html">PORTFOLIO</a>
+		</nav>
+
+		<hr>
+
+		<h1 class="head1">MEGAN GARTNER</h1>
+
```

Comment:
Nice work using class, but I wouldn't include that if you're not planning to style that element differently.